migration/src/main/resources/migrations/ensure-forms-consider-practicing-providers.xml

```diff
@@ -0,0 +1,78 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.8.xsd">
+
+    <!-- NOTE: This has to come before the setting of all users to be providers in this migration process. -->
+
+    <!-- First, we ensure that a role exists to associate with Providers to be selected on forms -->
+
+    <changeSet id="rwandaemr-ensure-practicing-provider-role-exists" author="IMB">
+        <preConditions onFail="MARK_RAN">
+            <sqlCheck expectedResult="0">
+                SELECT count(*) FROM role WHERE role = "Practicing provider"
+            </sqlCheck>
+        </preConditions>
+        <sql>
+            INSERT INTO `role`(
+                role,
+                description,
+                uuid
+            )
+            VALUES (
+                "Practicing provider",
+                "Role to differentiate practicing providers from other user providers",
+                "36a4a54d-ab0a-11ea-8991-3c6aa7d72242"
+            )
+            ;
```

Comment:
This is an area where we might need to at least consider how we are modeling providers in Rwanda going forward.  The standard in OpenMRS RefApp and PihCore is to use the ProviderManagement module (originally written for Rwanda ironically), which adds a "ProviderRole" attribute to Provider.  This is intentionally not the same as a "User Role", as User Roles are intended to be more about permissions and are attached to users, while Providers are not required to be linked to users.
There is an "<encounterProviderAndRole> tag in htmlformentry that utilizes this, and we'd presumably change all of our <encounterProvider> tags to <encounterProviderAndRole> tags.

@mogoodrich can you comment on this and help us to work out the approach you'd recommend here?